rustls/src/server/server_conn.rs

```diff
@@ -315,6 +315,10 @@ pub struct ServerConfig {
     /// If this is 0, no tickets are sent and clients will not be able to
     /// do any resumption.
     pub send_tls13_tickets: usize,
+
+    /// Whether require client to always send "extended_master_secret" extension
+    #[cfg(feature = "tls12")]
+    pub force_using_ems: bool,
```

Comment:
I propose the name `require_extended_master_secret`, or `require_ems`.

Also, I propose to include a link to the RFC in the doc comments:
```
If set to `true`, requires the client to support the extended master secret extraction method defined in [RFC 7627](https://datatracker.ietf.org/doc/html/rfc7627).

The default is `false`.
```

Reply:
Updated both server and client. `require_ems` is used now.